### PR TITLE
Update Christain's image

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: themes/webdriver.io/layout/_partial/footer.ejs [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <article class="col2">
             <p>This project is maintained by:</p>
             <a class="twitterLink" href="http://twitter.com/webdriverio" title="WebdriverIO"><img src="/images/twitter-profile.png" alt="@WebdriverIO"></a>
-            <a class="twitterLink" href="http://twitter.com/bromann" title="Christian Bromann"><img src="https://pbs.twimg.com/profile_images/1806512295/profil.png" alt="@bromann"></a>
+            <a class="twitterLink" href="http://twitter.com/bromann" title="Christian Bromann"><img src="https://avatars3.githubusercontent.com/u/731337?v=3&s=72" alt="@bromann"></a>
```
Comment (christian-bromann) [inline themes/webdriver.io/layout/_partial/footer.ejs]: How about https://pbs.twimg.com/profile_images/1806512295/profil_200x200.png

Comment (micahgodbolt) [inline themes/webdriver.io/layout/_partial/footer.ejs]: Still 58k. Could you upload a JPG instead of PNG. That should make a big difference.

**Comment by micahgodbolt** [comment]
50k is better than 500k :)

File: themes/webdriver.io/layout/_partial/footer.ejs (hunk 1 above)
Comment (christian-bromann) [inline themes/webdriver.io/layout/_partial/footer.ejs]: Let's tackle this in #42

**Comment by christian-bromann** [comment]
Thanks!

**Pull request merged**